src/components/TodoInput.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { TodoItem } from '../type';
+import * as localStore from '../utils/localStore'
+
+
+const TodoInput: React.FC = () => {
+    const [title, setTitle] = React.useState<string>('');
+
+    const onChangeTitle = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
    const onChangeTitle = (e: React.ChangeEvent<HTMLInputElement>):void => {
```
요기는 리턴타입 따로 표기 안한 이유가 있나용??밑에도

Reply:
사실 void면 따로 표기안해도 되는 거라서 사실 원래는 표기안하는데 ㅎㅎ; 연습 겸 걍 넣은부분이 있던겁니당. 그냥 통일하는겸 다 넣을게유 일단